devices.js

```diff
@@ -2001,6 +2001,10 @@ const devices = [
             fz.AC0251100NJ_cmdMoveToColorTemp, fz.AC0251100NJ_cmdMoveHue, fz.AC0251100NJ_cmdMoveToSaturation,
             fz.AC0251100NJ_cmdOff, fz.AC0251100NJ_cmdMove, fz.battery_3V,
             fz.AC0251100NJ_cmdMoveToLevelWithOnOff,
+            fz.AC0251700NJ_cmdOn, fz.AC0251700NJ_cmdMoveWithOnOff, fz.AC0251700NJ_cmdStop,
```

Comment:
I dont think extra converters are needed anymore, can you remove them?

Reply:
in devices.js and fromzigbee.js so ?

Reply:
https://github.com/Koenkk/zigbee-herdsman-converters/pull/768/commits/7bf685f51a6525bf2c62a4644448e1e52846129f